tika-parsers/tika-parsers-standard/tika-parsers-standard-modules/tika-parser-microsoft-module/src/main/java/org/apache/tika/parser/microsoft/ooxml/xwpf/XWPFStylesShim.java

```diff
@@ -43,9 +43,9 @@ public class XWPFStylesShim {
     /**
      * Empty singleton to be used when there is no style info
      */
-    public static XWPFStylesShim EMPTY_STYLES = new EmptyXWPFStyles();
+    public static final XWPFStylesShim EMPTY_STYLES = new EmptyXWPFStyles();
```

Comment:
this might be dangerous - as the field is public - someone could set the value to a non empty set of styles